<?xml version="1.0"?>
<launch>

  <include file="$(find lively_ik)/launch/load_info_file.launch">
    <arg name="info_file_name" value="info_ur3e.yaml"/>
  </include>

  <include file="$(find lively_ik)/launch/relaxed_ik_julia.launch"/>

</launch>
